samples/csharp_dotnetcore/17.multilingual-bot/README.md

```diff
@@ -2,6 +2,8 @@
 
 Bot Framework v4 multilingual bot sample
 
+This sample will present you with a set of cards to select your choice of language. You can either change language by invoking the option cards, or by entering the language code (en, es). The sample will then acknowledge your selection.
```

Comment:
```suggestion
This sample will present you with a set of cards to select your choice of language. You can either change language by invoking the option cards, or by entering the language's code (e.g, _en_, _es_).  Then, the bot will then acknowledge your selection.
```

Reply:
(For both readme.md)

Reply:
I have to remove 1 `then` from that last sentence